packages/ui-drilldown/src/Drilldown/README.md

```diff
@@ -181,6 +514,242 @@ const contactsData = [
+                Option
+              </Drilldown.Option>
+            </Drilldown.Page>
+
+            <Drilldown.Page
+              id="secondPage"
+              renderTitle={showTitle && 'Second page'}
+              renderActionLabel={showAction && 'Action!'}
+              renderBackButtonLabel={showAlternativeBackLabel
+                ? ((prevPageTitle) => prevPageTitle ? `Back to ${prevPageTitle}` : 'Back')
+                : undefined}
+            >
+              <Drilldown.Option id="option1">Option</Drilldown.Option>
+              <Drilldown.Option id="option2">Option</Drilldown.Option>
+              <Drilldown.Option id="option3">Option</Drilldown.Option>
+            </Drilldown.Page>
+          </Drilldown>
+        </Flex.Item>
+
+        <Flex.Item padding="0 0 0 large">
+          <FormFieldGroup description="Drilldown Group example settings">
+            <Checkbox
+              checked={this.state.showTitle}
+              label="Show page title"
+              variant="toggle"
+              onChange={
+                () => {
+                  this.setState({ showTitle: !this.state.showTitle })
+                }
+              }
+            />
+            <Checkbox
+              checked={this.state.showAction}
+              label="Show action label"
+              variant="toggle"
+              onChange={
+                () => {
+                  this.setState({ showAction: !this.state.showAction })
+                }
+              }
+            />
+            <Checkbox
+              checked={this.state.showAlternativeBackLabel}
+              label="Show alternative back navigation label"
+              variant="toggle"
+              onChange={
+                () => {
+                  this.setState({ showAlternativeBackLabel: !this.state.showAlternativeBackLabel })
+                }
+              }
+            />
+          </FormFieldGroup>
+        </Flex.Item>
+      </Flex>
+    )
+  }
+}
+
+render(<GroupsExample />)
+```
+
+### Option Groups
+
+Wrapping the Options in `<Drilldown.Group>` will separate these options with separators. These separators can be hidden, and you can provide a label with the `renderGroupTitle` prop.
+
+```js
+---
+example: true
+render: false
+---
+class GroupsExample extends React.Component {
+  state = {
+    showSeparators: true,
+    showTitles: true
+  }
+
+  render() {
+    return (
+      <Flex alignItems='start'>
+        <Flex.Item>
+          <Drilldown rootPageId='root' width="20rem" maxHeight="30rem">
+            <Drilldown.Page id="root">
+              <Drilldown.Group
+                id="group1"
+                renderGroupTitle={this.state.showTitles && 'Group'}
+                withoutSeparators={!this.state.showSeparators}
+              >
+                <Drilldown.Option id="item11">GroupItem</Drilldown.Option>
+                <Drilldown.Option id="item12">GroupItem</Drilldown.Option>
```

Comment:
I'd add some better placeholder text inside the Options, now its a bit hard to see whats happening. e.g. fruit names, countries, famous composers,...

Reply:
Done